### ISSUE TITLE
[Issue]: ID3 Tags Not Honored

### ISSUE THREAD

**Issue by unarj** [jellyfin/jellyfin#7224]
### Please describe your bug

this was working at some point as I have many albums that don't have other sources for metadata (self published, demo or promo, live shows, etc) and ones added previously are showing fine.

I was running docker/stable which is 10.7.7, recently updated to docker/unstable (10.8.0) to test, same result.

stored (`<artist>/<album>/<track>`), recently added albums with full ID3 tags embedded but no other metadata sources show up in Jellyfin with only album name and file names for track list.

when doing a metadata refresh, force overwrite, console output attached, 'Discarding ID3 tags because more suitable tags were found.' seems to be the relevant bit

### Jellyfin Version

10.7.7

### Environment

```markdown
- OS: ubuntu 20.04
- Virtualization: docker
- Clients: windows, android
- Browser: chrome, firefox
- FFmpeg Version: 
- Playback Method:
- Hardware Acceleration: i5
- Plugins:
- Reverse Proxy:
- Base URL:
- Networking:
- Storage:
```

### Jellyfin logs

```shell
jellyfin        | [mov,mp4,m4a,3gp,3g2,mj2 @ 0x55621c6586c0] Discarding ID3 tags because more suitable tags were found.
jellyfin        | [aac @ 0x55621c65bdc0] Prediction is not allowed in AAC-LC.
jellyfin        | [mov,mp4,m4a,3gp,3g2,mj2 @ 0x556b9821e6c0] Discarding ID3 tags because more suitable tags were found.
jellyfin        | [aac @ 0x556b98221dc0] Sample rate index in program config element does not match the sample rate index configured by the container.
jellyfin        | [aac @ 0x556b98221dc0] Inconsistent channel configuration.
jellyfin        | [aac @ 0x556b98221dc0] get_buffer() failed
jellyfin        | [mov,mp4,m4a,3gp,3g2,mj2 @ 0x555eec6fe6c0] Discarding ID3 tags because more suitable tags were found.
jellyfin        | [aac @ 0x555eec701dc0] Multiple frames in a packet.
jellyfin        | [aac @ 0x555eec701dc0] channel element 2.9 is not allocated
jellyfin        | [mov,mp4,m4a,3gp,3g2,mj2 @ 0x56312d9626c0] Discarding ID3 tags because more suitable tags were found.
jellyfin        | [aac @ 0x56312d965dc0] Sample rate index in program config element does not match the sample rate index configured by the container.
jellyfin        | [aac @ 0x56312d965dc0] Inconsistent channel configuration.
jellyfin        | [aac @ 0x56312d965dc0] get_buffer() failed
jellyfin        | [mov,mp4,m4a,3gp,3g2,mj2 @ 0x5630e3e556c0] Discarding ID3 tags because more suitable tags were found.
jellyfin        | [aac @ 0x5630e3e58dc0] channel element 2.9 is not allocated
jellyfin        | [mov,mp4,m4a,3gp,3g2,mj2 @ 0x55f6f8d3b6c0] Discarding ID3 tags because more suitable tags were found.
jellyfin        | [aac @ 0x55f6f8d3edc0] Reserved bit set.
jellyfin        | [aac @ 0x55f6f8d3edc0] TNS filter order 17 is greater than maximum 12.
jellyfin        | [mov,mp4,m4a,3gp,3g2,mj2 @ 0x560397fb56c0] Discarding ID3 tags because more suitable tags were found.
jellyfin        | [aac @ 0x560397fb8dc0] Sample rate index in program config element does not match the sample rate index configured by the container.
jellyfin        | [aac @ 0x560397fb8dc0] Inconsistent channel configuration.
jellyfin        | [aac @ 0x560397fb8dc0] get_buffer() failed
jellyfin        | [mov,mp4,m4a,3gp,3g2,mj2 @ 0x561629abf6c0] Discarding ID3 tags because more suitable tags were found.
jellyfin        | [aac @ 0x561629ac2c00] channel element 3.1 is not allocated
jellyfin        | [mov,mp4,m4a,3gp,3g2,mj2 @ 0x55a1904ce6c0] Discarding ID3 tags because more suitable tags were found.
jellyfin        | [aac @ 0x55a1904d1dc0] channel element 2.15 is not allocated
jellyfin        | [mov,mp4,m4a,3gp,3g2,mj2 @ 0x55a53db286c0] Discarding ID3 tags because more suitable tags were found.
jellyfin        | [aac @ 0x55a53db2bdc0] invalid band type
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by unarj** [comment]
this is still an issue, music album with no metadata source but fully populated MP3 tags shows only album title.

**Comment by unarj** [comment]
this appears to have been fixed with 10.8.0 docker image
